libs/ml/tests/unit/layers/scaled_dot_product_attention.cpp

```diff
@@ -50,13 +50,13 @@ TYPED_TEST(ScaledDotProductAttention, input_output_dimension_check)  // Use the
       "ScaledDotProductAttention", {query, key, value, mask}, static_cast<SizeType>(4),
       DataType(0.1));
   TypeParam query_data = TypeParam({4, 7, 2});
-  query_data.Fill(static_cast<DataType>(0.1));
+  query_data.Fill(fetch::math::Type<DataType>("0.1"));
   TypeParam key_data = TypeParam({4, 5, 2});
-  key_data.Fill(static_cast<DataType>(0.2));
+  key_data.Fill(fetch::math::Type<DataType>("0.2"));
   TypeParam value_data = TypeParam({3, 5, 2});
-  value_data.Fill(static_cast<DataType>(0.3));
+  value_data.Fill(fetch::math::Type<DataType>("0.3"));
   TypeParam mask_data = TypeParam({1, 7, 2});
-  mask_data.Fill(static_cast<DataType>(1));
+  mask_data.Fill(fetch::math::Type<DataType>("1"));
```

Comment:
below you have Datatype{1}, I think for consistency all integers should just be like this and avoid using string.

Reply:
In this code block above there are 3 more usage of DataType init, so I did it to keep consistent code.